admin/environment-management/cvms.md

```diff
@@ -52,46 +53,46 @@ gcloud beta container clusters create "YOUR_NEW_CLUSTER" \
 
 ### Azure Kubernetes Service
 
-If you're using Kubernetes version 1.18, Azure defaults to the correct Ubuntu node
-base image. When creating your cluster, set `--kubernetes-version` to `1.18.x`
-or newer for CVMs.
+If you're using Kubernetes version 1.18, Azure defaults to the correct Ubuntu
+node base image. When creating your cluster, set `--kubernetes-version` to
+`1.18.x` or newer for CVMs.
 
 ### Amazon Web Services w/ EKS
 
 You can modify an existing [AWS-hosted container](../../setup/kubernetes/aws.md)
-to support CVMs by [creating a
-nodegroup](https://eksctl.io/usage/managing-nodegroups/#creating-a-nodegroup-from-a-config-file)
+to support CVMs by
+[creating a nodegroup](https://eksctl.io/usage/managing-nodegroups/#creating-a-nodegroup-from-a-config-file)
 and updating your `eksctl` config spec.
 
 1. Define your config file in the location of your choice (we've named the file
    `coder-node.yaml`, but you can call it whatever you'd like):
 
-    ```yaml
-    apiVersion: eksctl.io/v1alpha5
-    kind: ClusterConfig
+   ```yaml
+   apiVersion: eksctl.io/v1alpha5
+   kind: ClusterConfig
 
-    metadata: 
-      version: "1.17"
-      name: <YOUR_CLUSTER_NAME>
-      region: <YOUR_AWS_REGION>
+   metadata:
+     version: "1.17"
+     name: <YOUR_CLUSTER_NAME>
+     region: <YOUR_AWS_REGION>
 
-    nodeGroups:
-    - name: coder-node-group
-      amiFamily: Ubuntu1804
-    ```
+   nodeGroups:
+     - name: coder-node-group
+       amiFamily: Ubuntu1804
+   ```
 
 1. Create your nodegroup (be sure to provide the correct file name):
 
-    ```console
-    eksctl create nodegroup --config-file=coder-node.yaml
-    ```
+   ```console
+   eksctl create nodegroup --config-file=coder-node.yaml
+   ```
```

Comment:
This fixed the look and feel :nerd_face: 

## Before

![Screenshot from 2021-03-23 09-50-57](https://user-images.githubusercontent.com/11184711/112166087-53721f80-8bbd-11eb-8b0a-1bc3cbd40a6e.png)


## After

![Screenshot from 2021-03-23 09-51-04](https://user-images.githubusercontent.com/11184711/112166106-5705a680-8bbd-11eb-8078-fd579d220990.png)

## Note

My process for reviewing these lint/format changes are opening the page to a fragment in 2 separate tabs

1. Live: https://coder.com/docs/admin/environment-management/cvms#amazon-web-services-w-eks
2. Preview: https://codercom-lglwr8awa-codercom.vercel.app/docs/admin/environment-management/cvms#amazon-web-services-w-eks

And flip back and forth inspecting where any differences occur.

Reply:
I wonder if we can make a page that iframes both side-by-side and links the vertical scroll... would be really helpful for visual diffs :thinking: Maybe a hackathon project!